data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+					goto who
+				`	"What... What is this feeling?"`
+			`	There is a pause. The feeling of being watched becomes intense, as if you're being studied intently.`
+			`	"A human would call it... dread. Regret. Loss. You are standing in the graveyard of a lost civilization."`
+			label who
+			`	"I am the guardian of this site. I am also the one who guided you towards the buried city, so your species would bear witness and take heed of the lessons of ones who have long succumbed to their own folly. I did not intend for you to reach this far."`
+			`	"Here is where the Sheragi came to an end. The tools that drove them to extinction are long gone. We witnessed as they cast those tools into the heart of this star."`
+			choice
+				`	"You mean this ship? Its weapons systems seem to be intact."`
+					goto cannon
+				`	"What tools?"`
+			`	"Ones capable of crippling an entire world's worth of life."`
+				goto next
+			label cannon
+			`	"That ship is just a relic. The weapon that perished the Sheragi was indiscriminate. An irrational decision they made themselves."`
+			label next
+			`	As that is said, you're overcome by glimpses of memories that are not yours, not clear enough for you to identify what exactly is happening, but you can feel the sadness in them.`
+			`	The voice continues: "I know what you seek here. The trinket you covet is of little relevance, even for your own, human standards. But if you wish to possess it, then take it. Perhaps the sight of that vessel will remind the living races of those who have come and gone... and how, despite their great accomplishments, their decision to make war reduced them to graveyards, relics, and memories."`
+			`	The voice, and the overwhelming emotions that came alongside it, disappear as mysteriously as they came. Desyo, who witnessed you speaking into thin air, asks whether you feel okay. It seems like no one else heard it.`
+
+	npc board
+		government "Sheragi"
+		personality derelict mute
+		system Zubenelhakrabi
+		ship "Emerald Sword" "Emerald Sword"
+		conversation
+			`You dock with the Emerald Sword, and the crew starts working to carefully find a way in, directed by the Hai archaeologists. There is no atmosphere inside it as the life support has long since ceased to work, and it takes a long time to restore it through your ship's own life support and patch tiny micrometeorite holes. Once it's in breathable conditions, it still smells like a tomb. You find many Sheragi skeletons and mummified remains. A few of them have prosthetics that survived better than their host. Your crew gets to work cleaning up the thick layer of dust and grime that covers every surface, while the more specialized crew tries to bring the power core back to life and take advantage of the zero-G environment to move the jump drive into the ship more easily. You'll need to land to properly install it, however. All that is left for you to do is relocate the many corpses littering the living spaces with your crew.`
```

Comment:
I feel like you could really extend this conversation to make it more impactful. After having the Archon speak to you, I feel like it'd be fitting to linger on the skeletons here for much longer. Perhaps you have the option to wander the ship and observe the bodies; I imagine there are stories to be told by the positions that the Sheragi died in. And during that time the player could perhaps speak with Foster and ask if what you're doing is wrong; you're basically tomb raiding right now, after all.

Reply:
Interesting idea!

Reply:
Seconded, huge potential for emotionally impactful narrative.